ovs/lib/update.py

```diff
@@ -124,17 +128,25 @@ def _get_package_information_core(client, package_info):
                 storagedriver_services.append('dtl_{0}'.format(sd.vpool.name))
                 storagedriver_services.append('volumedriver_{0}'.format(sd.vpool.name))
 
+            alba_package = 'alba' if 'alba' in installed.keys() else 'alba-ee'
+            volumedriver_base_package = 'volumedriver-no-dedup-base' if 'volumedriver-no-dedup-base' in installed.keys() else 'volumedriver-ee-base'
+            volumedriver_server_package = 'volumedriver-no-dedup-server' if 'volumedriver-no-dedup-server' in installed.keys() else 'volumedriver-ee-server'
+
+            version_mapping = {'volumedriver-server': ['volumedriver-no-dedup-server', 'volumedriver-ee-server'],
```

Comment:
This is expected right?  Twice volumedriver-server package io volumedriver-base? Because i guess we check only the volumedriver-server package at this point?